gradio_app.py

```diff
@@ -610,4 +610,4 @@ def get_app():
     MODELS_PATH = args.models_path
 
     app = get_app()
-    app.queue().launch(share=args.share, server_port=args.port)
+    app.queue().launch(share=args.share, server_name="0.0.0.0", server_port=args.port)
```

Comment:
Thanks! could you explain what difference does server_name make? or maybe some links I can read up

Reply:
server_name ="0.0.0.0" make the gradio be accessible from other machine in local network, instead of sharing to the public URL. in case your college need to access the gradio app within the local network.

Reply:
Thanks, I'll still default it to 127.0.0.1 and add a comment to let people know they can set it to 0.0.0.0 to let other machines in the local network to access it. This is for security purposes.